src/app/frontend/replicasetdetail/replicasetdetail_controller.js

```diff
@@ -181,4 +205,46 @@ export default class ReplicaSetDetailController {
    * @private
    */
   onReplicaSetDeleteError_(err) { this.log_.error(err); }
+
+  /**
+   * Applies sorting on pods table.
+   * @param {string} sortBy
+   * @export
+   */
+  sortPods(sortBy) {
```

Comment:
I see that there is no logic now in the controller. This means no tests :) You don't need to to test templates and ng-sort there